common/options.go

```diff
@@ -79,7 +90,11 @@ func BuildUrlWithBlockQueryOptions(path string, options BlockQueryOptions) strin
 // AccountQueryOptions holds options for account queries
 type AccountQueryOptions struct {
 	OnFinalBlock   bool
-	OnStartOfEpoch uint32
+	OnStartOfEpoch core.OptionalUint32
```

Comment:
[`api.AccoutQueryOptions`](https://github.com/ElrondNetwork/elrond-go-core/blob/main/data/api/options.go#L6) was not referenced / used, [intentionally](https://softwareengineering.stackexchange.com/a/366237). All right?